pom.xml

```diff
@@ -45,22 +68,12 @@
         <dependency>
             <groupId>org.liquibase</groupId>
             <artifactId>liquibase-core</artifactId>
-            <version>3.3.5</version>
+            <version>${liquibase.version}</version>
```

Comment:
I see that we are excluding 
```
    <exclusions>
                <exclusion>
                    <groupId>javax.servlet</groupId>
                    <artifactId>jstl</artifactId>
                </exclusion>
```
from liquibase-core in cc connector. Have you checked if we still need to exclude that?

Reply:
I haven't check that, thanks.
So, running `mvn dependency:tree -Dverbose` and searching for `jstl` I was unable to find anything.
We have only `javax.servlet:javax.servlet-api:jar:3.1.0:compile`.
I checked in cc connector as well: there is no `jstl` dependency.
As discussed may be this dependency was in previous versions, that is why I will skip adding it as an `exclusion`
Thanks